src/allotropy/parsers/lines_reader.py

```diff
@@ -132,19 +134,15 @@ def pop_csv_block_as_lines(self, empty_pat: str = EMPTY_STR_PATTERN) -> list[str
     def pop_csv_block_as_df(
         self,
         empty_pat: str = EMPTY_STR_PATTERN,
-        *,
```

Comment:
What is the purpose of removing this?

Reply:
this is the same idea as in lines_as_df. To stop ignoring parameters and instead allow the function to pass kwargs directly to pandas.